src/Model/MethodGo.cs

```diff
@@ -33,6 +33,8 @@ public class MethodGo : Method
         public bool IsCustomBaseUri
             => CodeModel.Extensions.ContainsKey(SwaggerExtensions.ParameterizedHostExtension);
 
+        public bool RegisterRP;

```

Comment:
Mind either renaming this, or adding a comment making clear that this controls whether or not a `RegisterRPSenderDecorator` will be added? If it were renamed, maybe something like `DecorateUsingRegisterRP`? I know that's not a very concise name, so I would totally understand just commenting.

Reply:
Fixed